app/views/auctions/_bids_and_form.html.erb

```diff
@@ -15,3 +15,7 @@
     </span>
   </div>
 </div>
+
+<div class="issue-bids-info-item">
+  <%= render partial: auction.bid_button_partial, locals: { auction: auction.auction } %>
```

Comment:
yes it does look weird! I delegated `id` in the view model so we can do named routes. will use `auction.id` instead